src/app/pages/listing/listing-edit-upload.jsx

```diff
@@ -5,13 +5,15 @@ import {
   CardContent,
   CircularProgress,
   Container,
+  Fade,
   FormControlLabel,
+  List,
   Switch,
   Typography,
   makeStyles,
 } from '@material-ui/core';
+import HelpOutlineIcon from '@material-ui/icons/HelpOutline';
 import { number, oneOfType, string } from 'prop-types';
-
```

Comment:
Removing this line puts an import of a "nearby" component next to the set of imports from external components. I think our linter should be mad at this